types/@hummer/index.d.ts

```diff
@@ -0,0 +1,248 @@
+// Type definitions for hummer 1.0
```

Comment:
the directory name should be `hummer` not `@hummer` to match the package name.

Reply:
Hi sandersn, as mentioned in a previous comment, this is a definition for '@hummer' not 'hummer', please check the previous comment for more details.